front/app/containers/Authentication/useSteps/stepConfig/sharedSteps.ts

```diff
@@ -167,6 +167,9 @@ export const sharedSteps = (
         if (error_code === 'not_entitled_under_minimum_age') {
           setCurrentStep('missing-data:verification');
           setError('not_entitled_under_minimum_age');
+        } else if (error_code === 'taken') {
+          setCurrentStep('missing-data:verification');
+          setError('verification_taken');
```

Comment:
I guess this is the part that we could replace by the new `access-denied` step that I added on my other PR